src/Qandidate/Common/Symfony/HttpKernel/EventListener/JsonRequestTransformerListener.php

```diff
@@ -32,14 +34,14 @@ public function onKernelRequest(GetResponseEvent $event)
         }
 
         if (! $this->transformJsonBody($request)) {
-            $response = Response::create('Unable to parse request.', Response::HTTP_BAD_REQUEST);
+            $response = Response::create('Unable to parse request.', 400);
             $event->setResponse($response);
         }
     }
 
     private function isJsonRequest(Request $request)
     {
-        return 'json' === $request->getContentType();
+        return in_array($request->getContentType(), $this->jsonContentTypes);
```

Comment:
Symfony actually returns `json` here.
See [getContentType], [getFormat] and [initializeFormats].

[getContentType]: https://github.com/symfony/symfony/blob/7ebda1e155c3f5e81053eb0c7fc1f38ae2af1eda/src/Symfony/Component/HttpFoundation/Request.php#L1380-L1383
[getFormat]: https://github.com/symfony/symfony/blob/7ebda1e155c3f5e81053eb0c7fc1f38ae2af1eda/src/Symfony/Component/HttpFoundation/Request.php#L1303-L1318
[initializeFormats]: https://github.com/symfony/symfony/blob/7ebda1e155c3f5e81053eb0c7fc1f38ae2af1eda/src/Symfony/Component/HttpFoundation/Request.php#L1836